starfish/image/_stack.py

```diff
@@ -83,9 +84,22 @@ def __init__(self, image_partition):
             if ix == axis_data.order
         ]
         shape.extend(self._tile_shape)
+        dims = [
+            axis_name.value
+            for axis_name, axis_data in ImageStack.AXES_DATA.items()
+            for ix in range(ImageStack.N_AXES)
+            if ix == axis_data.order
+        ]
+        dims.extend([Coordinates.Y.value, Coordinates.X.value])
```

Comment:
Coordinates. 🤢 (No change necessary, will fix in PR-to-be following your stringification of enums.)